app/views/patients/show.html.erb

```diff
@@ -115,8 +115,12 @@
           <th class="DataTable-table-header">Last Name</th>
           <th class="DataTable-table-header">Phone Number</th>
           <th class="DataTable-table-header">Email</th>
+          <th class="DataTable-table-header">Last Date of Exposure</th>
+          <th class="DataTable-table-header">Assigned User</th>
+          <th class="DataTable-table-header">Notes</th>
           <th class="DataTable-table-header">Contact Attempts</th>
-          <th class="DataTable-table-header">Actions</th>
+          <th class="DataTable-table-header">Enrolled</th>
```

Comment:
The ticket specifies that this column should be "Enrolled?" with a question mark.

Reply:
Corrected.